README.md

```diff
@@ -106,6 +106,16 @@ respectively.  This should not be used in production.
 }
 ```
 
+##### disabled     - optional
+
+If `true` `oErrors` will be configure with no operation functions and errors
+will not be reported.
```

Comment:
I consciously chose disabled for a couple of reasons, the main reason is by default I want it to be enabled, and disabled only by explicitly setting `disabled = true`. I'm happy to change as long as the default is still on when `options.enabled` is undefined.